### DISCUSSION TITLE
[CLOSED] Où afficher le bouton d'export des boutons/panneaux

### DISCUSSION THREAD

**Discussion by Arubinu** [skarab42/marv#187] [General]
Bonjour,

Du coup j'ai trouvé bizarre d'afficher le bouton d'export uniquement lors du clique droit sur un bouton et non dans l'espace vide.
Alors que le bouton d'import est disponible dans les deux cas.

Dans l'état actuel, on a l'impression que c'est uniquement le bouton qui sera exporté ^^

**Comment by skarab42** [comment]
Hello, c'est le cas, tu peu exporter un bouton ou un panneau selon la ou tu clique ;)

**Reply by Arubinu** [reply]
Ah ok, donc un bouton ou un panneau.

Je vois je vois.
